packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -1140,6 +1140,23 @@ describeCompat("stashed ops", "2.0.0-rc.1.0.0", (getTestObjectProvider, apis) =>
 		);
 	});
 
+	it("fails when session time expires using stashed time", async function () {
+		const pendingOps = await getPendingOps(provider, false, async (c, d) => {
+			const map = await d.getSharedObject<SharedMap>(mapId);
+			[...Array(lots).keys()].map((i) => map.set(i.toString(), i));
+		});
+		const pendingState = JSON.parse(pendingOps);
+		assert.ok(pendingState.pendingRuntimeState.sessionExpiryTimerStarted);
+		pendingState.pendingRuntimeState.sessionExpiryTimerStarted = 1;
+		const pendingOps2 = JSON.stringify(pendingState);
+		try {
+			await loader.resolve({ url }, pendingOps2);
+			assert.fail("should have failed");
+		} catch (error: any) {
+			assert.strictEqual(error.message, `Client session expired.`);
+		}
```

Comment:
In case you're not familiar with `assert.rejects`, FYI you can do it this way too:
```suggestion
		await assert.rejects(async () => loader.resolve({ url }, pendingOps2), /Client session expired./);
```